sources/packages/api/src/services/student-appeal-request/student-appeal-request.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from "@nestjs/common";
+import { RecordDataModelService } from "../../database/data.model.service";
+import { Connection, SelectQueryBuilder } from "typeorm";
+import {
+  StudentAppealRequest,
+  StudentAppealStatus,
+} from "../../database/entities";
+
+@Injectable()
+export class StudentAppealRequestsService extends RecordDataModelService<StudentAppealRequest> {
+  constructor(connection: Connection) {
+    super(connection.getRepository(StudentAppealRequest));
+  }
+  /**
+   * Creates a 'select' query that could be used in an 'exists' or
+   * 'not exists' where clause.
+   * @param appealStatus student request status that to be searched for.
```

Comment:
Please review the text. I believe that the idea was to have something like _student request status to be searched for._.